src/Components/Web.JS/src/Platform/WebAssemblyProgressReporter.ts

```diff
@@ -0,0 +1,21 @@
+import { WebAssemblyProgressService } from "./WebAssemblyProgressService";
+
+export class WebAssemblyProgressReporter {
+    static setProgress(resourcesTotal: number, resourcesLoaded: number): void {
+        const circle = document.querySelector('.progress') as SVGCircleElement;
+        const circumference = 2 * Math.PI * circle.r.baseVal.value;
+        const ring = circumference - resourcesLoaded / resourcesTotal * circumference;
+        circle.style.strokeDasharray = circumference.toString() + " " + circumference.toString();
+        circle.style.strokeDashoffset = ring + '';
```

Comment:
Is the `+ ''` here because `strokeDashoffset` expects a string?

Reply:
Yes. I have changed it to use the toString() method.